test/experiment.test.ts

```diff
@@ -81,5 +81,11 @@ describe('LocalExperiment', () => {
 			exp.engage('test-device-1');
 			expect(identifyCallsCount).toStrictEqual(1);
 		});
+
+		it('sets user property on re-engage', () => {
```

Comment:
This test may be run in parallel with the previous one. Let's better move the second call directly to the original test:
```
exp.engage('test-device-1');
expect(identifyCallsCount).toStrictEqual(1);
exp.engage('test-device-1');
expect(identifyCallsCount).toStrictEqual(2);
```